src/logs.py

```diff
@@ -1,28 +1,42 @@
+# This code imports the `glob`, `os`, and `time` modules
 import glob
 import os
 import time
 
+# The `Logging` class is defined
 class Logging:
+    # The class constructor initializes the `logFileOpened` attribute as `False`
     def __init__(self):
         self.logFileOpened = False
 
-    def log(self, stringToLog: str):
-        # creating logs folder
-        try:
-            os.mkdir(os.getcwd() + "\logs")
-        except FileExistsError:
-            pass
-        filenames = []
-        for filename in glob.glob(r"logs/log-*.txt"):
-            filenames.append(int(filename[9:-4]))
-        if len(filenames) == 0:
-            filenames.append(0)
-        if self.logFileOpened:
-            with open(f"logs/log-{max(filenames)}.txt", "a") as logFile:
-                logFile.write(f"[{time.strftime('%Y.%m.%d-%H.%M.%S', time.localtime(time.time()))}]"
-                              f" {stringToLog.encode('ascii', 'replace').decode()}\n")
-        else:
-            with open(f"logs/log-{max(filenames) + 1}.txt", "w") as logFile:
-                self.logFileOpened = True
-                logFile.write(f"[{time.strftime('%Y.%m.%d-%H.%M.%S', time.localtime(time.time()))}]"
-                              f" {stringToLog.encode('ascii', 'replace').decode()}\n")
+    # The `log` method takes a string `log_string` as an argument
+    def log(self, log_string: str):
+        # The logs directory is determined
+        logs_directory = os.getcwd() + "/logs"
+
+        # If the logs directory does not exist, it is created
```

Comment:
I'd argue this is unnecessary, `if not os.path.exists()` reads similarly to standard English.